### PR TITLE
Added Gradle plugin site config and published Gnag to plugin site

### LINKED ISSUE #38: Publish on Gradle plugin site

### PR DESCRIPTION
**Pull request by btkelly**
Resolves #38

### CODE REVIEW

File: plugin/bintray.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ publishing {
     publications {
         GnagPublication(MavenPublication) {
             from components.java
-            artifactId 'gnag'
+            artifactId gnagArtifcatId
```
Comment (stkent) [inline plugin/bintray.gradle]: Typo, `gnagArtifcatId` -> `gnagArtifactId`

File: plugin/gradle-site.gradle [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            tags = gnagLabels
+            version = gnagPluginVersion
+        }
+    }
+
+    mavenCoordinates {
+        artifactId = gnagArtifcatId
+        version = gnagPluginVersion
+    }
+}
```
Comment (stkent) [inline plugin/gradle-site.gradle]: Missing newline.

File: plugin/gradle-site.gradle [hunk 3]
```diff
@@ -0,0 +1,22 @@
+apply plugin: "com.gradle.plugin-publish"
+
+pluginBundle {
+
+    website = gnagWebsite
+    vcsUrl = gnagGitUrl
+    description = gnagDescription
+
+    plugins {
+        greetingsPlugin {
```
Comment (stkent) [inline plugin/gradle-site.gradle]: `greetings`, wuuuut?

**Review by stkent — CHANGES_REQUESTED** [review]
See inline comments

**Comment by btkelly** [comment]
@stkent Changes in 👍

**Review by stkent — APPROVED** [review]

**Pull request merged**